### DISCUSSION TITLE
网络不佳

### DISCUSSION THREAD

**Discussion by a8791881** [Meloong-Git/PCL#2204] [提问] [answered]
我是一名大学生 在学校的时候功能全都正常使用 回家之后在mod界面就出现了网络不佳的字样 但是其他游戏正常启动 后来通过正版验证登录的时候都无法启动 到现在离线登录显示需要补全文件 但是网络不佳 全文如下：
你的网络不佳 导致难以连接服务器 请检查网络 多重试几次 或尝试使用VPN
下面是：
基础连接已关闭：发送时发生错误
无法从传输连接中读取数据 远程主机关闭了一个现有的连接
在PCLModLaunch,McLaunchStart(LoaderTask2 Loader)
在PCLModNet.NetFile.Thread(NetThread Info)
错误类型: SystemNet.Sockets.SocketException
此类网络不佳 我在b站上尝试过部分方法 都未能有效 故希望解答一下 万分感谢

**Comment by Pigeon0v0** [comment]
挂梯子，没了

**Comment by Chiloven945** [comment, marked as answer]
梯子不行就换一个梯子，也有可能是你家里网络的问题，尝试更换运营商等方法
